packages/wordcount/src/defaultSettings.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * Internal dependencies
+ */
+import type { WPWordCountDefaultSettings } from './types';
+
+/**
+ * Default settings for word counting operations.
+ */
+export const defaultSettings: WPWordCountDefaultSettings = {
```

Comment:
Instead of using the type annotation here, we can use the `satisfies` keyword to remove the duplicated type. See the suggestion below.